Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -411,13 +411,20 @@ right at the XDP layer are not visible in tcpdump since packet taps come at a mu
 later stage in the networking stack. Cilium's monitor or metric counters can be used
 instead for gaining visibility.
 
-NodePort Device, Port and Bind settings
+NodePort Devices, Port and Bind settings
 ***************************************
 
 When running Cilium's BPF kube-proxy replacement, by default, a NodePort or
-ExternalIPs service will be accessible through the IP address of a native device
-which has the default route on the host. To change the device, set its name in
-the ``global.devices`` helm option.
+LoadBalancer service or a service with externalIPs will be accessible through
+the IP addresses of native devices which either have the default route on
+the host or have Kubernetes InternalIP or ExternalIP assigned. To change
```

Comment:
I would mention that these devices are auto-detected by Cilium agent. And then `To manually change [...]`

Reply:
It might be good to keep a few indication of how detection works, though I agree it should be after this paragraph.

Auto-magically is great until it stops working and then users will come asking why it doesn't behave as they expected. Having a doc. to point at would be best (unless it's already explained somewhere else?).